lib/capybara/spec/session/visit_spec.rb

```diff
@@ -77,6 +86,12 @@
       serverless_session.visit("http://#{@session.server.host}:#{@session.server.port}/foo")
       expect(serverless_session).to have_content("Another World")
     end
+
+    it "should visit an absolute URL without scheme" do
```

Comment:
Is this actually a valid test? Isn't this just testing that scheme defaults to http if none is specified, and is that really valid to do?